qark/plugins/logging_plugin.py

```diff
@@ -0,0 +1,117 @@
+import sys, os, re
+
+sys.path.insert(0, os.path.dirname(os.path.realpath(__file__)) + '../lib')
+
+from yapsy.IPlugin import IPlugin
+from plugins import PluginUtil
+from modules import common
+from lib.pubsub import pub
+import lib.plyj.parser as plyj
+import lib.plyj.model as m
+
+class LoggingIssuesPlugin(IPlugin):
+
+    debug_regex = r'Log\.d'
+    verbose_regex = r'Log\.v'
+
+    def target(self, queue):
+        # get all decompiled files that contains usage of WebView
+        files = common.java_files
+        global parser, tree, fileName, verbose, debug, string_filename_d, string_filename_v
+        parser = plyj.Parser()
+        len_d = []
+        len_v = []
+        len_filename_d = []
+        len_filename_v = []
+        res = []
+        tree = ''
+        count = 0
+        for f in files:
+            count += 1
+            pub.sendMessage('progress', bar=self.getName(), percent=round(count * 100 / len(files)))
+            fileName = str(f)
+            try:
+                tree = parser.parse_file(f)
+            except Exception as e:
+                continue
+            # Traverse down the tree to find out verbose or debug logs
+            try:
+                for type_decl in tree.type_declarations:
+                    if type(type_decl) is m.ClassDeclaration:
+                        for t in type_decl.body:
+                            if type(t) is m.MethodDeclaration:
+                                if str(t.name) == 'v':
+                                    len_v.append(str(t.name))
+                                    len_filename_v.append(fileName)
+                                elif str(t.name) == 'd':
+                                        len_d.append(str(t.name))
+                                        len_filename_d.append(fileName)
+            except Exception as e:
+                continue
+
+        # Join all the filename and path containing debug and verbose logging
+        string_filename_d = " \n".join(len_filename_d)
+        string_filename_v = " \n".join(len_filename_v)
+        queue.put(res)
+
+        if len(len_filename_d) > 0:
```

Comment:
This can be rewritten as `if len_filename_d:` also, as discussed, consider renaming these variables.